docs/manual/mustcall-checker.tex

```diff
@@ -0,0 +1,182 @@
+\htmlhr
+\chapterAndLabel{Must Call Checker}{mustcall-checker}
+
+The Must Call Checker conservatively over-approximates
+the set of methods that an object should call before it is de-allocated.
+It is intended to be run as a subchecker of another checker: on its own, it does not
+enforce any rules other than subtyping.
+
+For example, consider an object whose type is \<java.io.OutputStream>.
+This \<OutputStream> might have an obligation to call the \<close()> method, to release an
+underlying file resource. Or, the \<OutputStream> might not have such an obligation, if the
+underlying resource is a byte array. The Must Call Checker over-approximates the methods that this
+\<OutputStream> should call by assigning it the type \<@MustCall(\{"close"\}) OutputStream>, which can be read
```

Comment:
"this OutputStream" is ambiguous because the previous two sentences each introduces an OutputStream.
I suggest that after sentence 2, the manual should say that its type is `@MustCall("close")`.  After sentence 3, the manual should say that its type is `@MustCall({})`.  Then just after that (before this sentence), say that you are talking about an OutputStream expression that might evaluate to either type of OutputStream, or say that you are talking about an arbitrary OutputStream.

Reply:
Thanks - I think this suggestion makes the paragraph a lot more clear.